examples/config-sync.json

```diff
@@ -0,0 +1,52 @@
+{
+	"version":"0.1.0-dev",
+	"storage":{
+		"rootDirectory":"/tmp/zot"
+	},
+	"http":{
+		"address":"127.0.0.1",
+		"port":"5000"
+	},
+	"log":{
+		"level":"debug"
+	},
+	"extensions":{
+		"sync":[
+			{
+				"url": "https://registry1:5000",
+				"pollInterval": "6h",
+				"tlsVerify": true,
+				"credentials": {
+					"username": "user",
+					"password": "pass"
+				},
+				"certDir": "/home/user/certs",
+				"content":[
+					{
+						"prefix":"repo1/repo",
+						"tags":{
+							"regex":"4.*",
```

Comment:
So this means get me all semver tags starting with 4.*

Reply:
Yes. it's an intersection.